### PR TITLE
admin create account page: data format can be misleading #5162

### LINKED ISSUE #5162: admin create account page: data format can be misleading
These brackets can be misleading (the actual format does not require brackets)
image

### PR DESCRIPTION
**Pull request by yasirunilan**
Fixes #5162

### CODE REVIEW

**Comment by thyageshm** [comment]
@yasirunilan is this pr ready for review?

**Comment by bamboo3250** [comment]
@thyageshm Is it okay to have 2 PRs fix the same issue? Since we have already had #5166.

**Comment by damithc** [comment]
Not recommended, but allowed. We'll merge whichever PR that is ready to merge first.

**Comment by yasirunilan** [comment]
Yeah ready. @thyageshm

File: .launches/All tests.launch [hunk 1]
```diff
@@ -9,12 +9,15 @@
 <stringAttribute key="org.eclipse.jdt.launching.VM_ARGUMENTS" value="-Duser.timezone=UTC-Xss2m"/>
 <mapAttribute key="org.testng.eclipse.ALL_CLASS_METHODS"/>
 <listAttribute key="org.testng.eclipse.CLASS_TEST_LIST"/>
+<booleanAttribute key="org.testng.eclipse.DEBUG" value="false"/>
```
Comment (bamboo3250) [inline .launches/All tests.launch]: Are these changes necessary? If not, please undo them.

**Comment by bamboo3250** [comment]
@yasirunilan after changing the `.tag` file, we need to run godmode to update html output of the page. Please check https://github.com/TEAMMATES/teammates/blob/master/devdocs/godmode.md for how to run godmode.

**Comment by wkurniawan07** [comment]
Hi @yasirunilan, any progress on this PR? It'll be great if you can finish this soon. If you can't finish it by this milestone, let's push it to the next one. However, if you abandon it for too long, I'm afraid I have to close it.

**Comment by yasirunilan** [comment]
I will try to do it by this milestone

**Comment by wkurniawan07** [comment]
Closing due to inactivity.
